### ISSUE TITLE
Myntra Fashion Product Analysis using Image Processing

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#413]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Myntra Fashion Product Analysis using Image Processing</br>
:red_circle: **Aim** : The aim of this project is to analyze the images of the fashion products using image processing methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/djagatiya/myntra-fashion-product-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Adm-2005** [comment]
Full name : Akshat Mishra
GitHub Profile Link : https://github.com/Adm-2005
Email ID : akshatdmishra2005@gmail.com
Approach for this Project : Keras Sequential Model with 3 additional layers.
What is your participant role? : Swoc 24 contributor

**Comment by abhisheks008** [comment]
Can you come up with 2 more models which you want to implement?
@Adm-2005

**Comment by Adm-2005** [comment]
Sure. If I were to make a list the other two models would be ResNet and FashionNet. The reason keras sequential was my first choice was because FashionNet sometimes due to its specific purpose sometimes results in overfitting.

**Comment by AgrawalTitiksha** [comment]
Full name : Titiksha Agrawal
GitHub Profile Link : https://github.com/AgrawalTitiksha
Email ID : agrawaltn2311@gmail.com
Approach for this Project : After data preprocessing and visualization, using PCA algorithm for normalizing and standardizing the images, then to use LDA , SVM (all 3 kernel) , and CNN (VGG16) for analyzing the images 
What is your participant role? (Mention the Open Source program) : @Contributor 2024

**Comment by abhisheks008** [comment]
Cool, @Adm-2005 use all the three models for this project and make a comparison of the models based on the accuracy scores to find out the best fitted model for this dataset/project.

Issue assigned to you. You can start working on it.

**Comment by Adm-2005** [comment]
Sure, I'll do that.

**Comment by kyra-09** [comment]
Hey , @abhisheks008 

Full name : Aaradhya Singh
GitHub Profile Link : https://github.com/kyra-09
Email ID : aaradhyasinghgaur@gmail.com
Participant ID (if applicable):

Approach for this Project : 
1.) I want to use Keras API to implement various pre-trained models and optimise them to make better accuracy for mentioned dataset .

2.) Comparing between used models with various performance metrics such as - f1 score , accuracy , confusion matrix etc.


What is your participant role? (Mention the Open Source program) - Contributor/GSSOC-2024

**Kindly assign this issue to me**

**Comment by abhisheks008** [comment]
Hi @kyra-09 one issue at a time please!

**Comment by abhisheks008** [comment]
As you have previously contributed in the repository, can you find out a dataset with the same problem statement which will be compatible with the deep learning methods?

@CoderOMaster

**Comment by CoderOMaster** [comment]
Sorry my bad it has images, found out after downloading since kaggle didn't show overview.anyways I will use to classify what type.of.clothing accesories it is based on CNN, resnet,vgg etc models

**Comment by CoderOMaster** [comment]
Hey you didn't assign me the issue

**Comment by abhisheks008** [comment]
Oops! Sorry for that.

**Comment by CoderOMaster** [comment]
@abhisheks008 I am unable to find a conclusive result since it has huge no of labels to predict anything.at the moment I tried using NLP to predict whether product is expensive or not by providing description of that dress/product. Then I am using cv to use images with brands as labels from CSV but brand names are too huge to implement and get good accuracy.very poor results as brands are itself 1000+ to be categorised.Should I publish my results anyways?

**Comment by abhisheks008** [comment]
> @abhisheks008 I am unable to find a conclusive result since it has huge no of labels to predict anything.at the moment I tried using NLP to predict whether product is expensive or not by providing description of that dress/product. Then I am using cv to use images with brands as labels from CSV but brand names are too huge to implement and get good accuracy.very poor results as brands are itself 1000+ to be categorised.Should I publish my results anyways?

I'd like to suggest you something. Can you look for a better dataset than this which suits the existing problem statement.

**Comment by CoderOMaster** [comment]
@abhisheks008 I found something after extensive research.I hope this will be suffice although this dataset is also similar to that so used single column for classification that too which had over 36 classes
